File: lab4/zadanie2/python/main.py
import sys
from totient import totient

def main():
	if len(sys.argv) < 2:
		print("Podaj parametry!")
		return -1
	for i in sys.argv[1:]:
		result = totient(int(i))
		print(f"totient({i})={result}")

if __name__ == "__main__":
	main()

File: lab4/zadanie2/c/main.c
#include <stdio.h>
#include "totient.h" 

int main(int argc, char *argv[]){
  if (argc<2){
    printf("Podaj parametry!\n");
    return -1;
  }
  int n, result;
  for (int i=1; i<argc; i++){
    sscanf(argv[i], "%u", &n);
    result = totient(n);
    printf("totient(%d)=%u", n, result);
  }

  return 0;
}
